lua/lvim/core/commands.lua

```diff
@@ -10,6 +10,8 @@ vim.cmd [[
   endfunction
 ]]
 
+vim.cmd [[command! LvimDocs execute 'silent !open "https://www.lunarvim.org/docs/installation"']]
```

Comment:
```suggestion
```
```lua
M.default = {
  -- snip
 {
    name = "LvimDocs",
    fn = function()
      vim.fn.execute [[!open "https://www.lunarvim.org/docs/installation"]]
    end,
  },
}
```

Reply:
Hey @cpea2506 👋 Thank you very much for your suggestion. That would fit much better in the existing structure. 

I’ll change and test this today! 🙂

Reply:
Maybe something like [this](https://github.com/dhruvasagar/vim-open-url/blob/66ed3caac15ed629ddc1740e416519fe3d71a4fb/autoload/open_url.vim#L3-L9) converted to lua and with `xdg-open` added so the command would be cross-platform

Reply:
Great idea @LostNeophyte 👌

Reply:
Hey @cpea2506 @LostNeophyte 👋 I have pushed a refactor based on your suggestions above. 

I have added OS reference targets which I think are applicable to the project but please correct me if I’ve got that wrong. I'm not 100% sure on requiring `silent`, it is preferrable to have that as a safe guard?

I’ve been able to successfuly test this change on macOS (Ventura 13.0) however I’m currently having issues with virtualising Windows & Linux on Apple Silicon. If there is a possibility of helping to test the change on those platforms that would be great; in the mean time I’ll continue trying to set them up.

I do have a small suggestion; would it be better for the documentation URL to point to the Quick Start page instead of the Installation page, given the user has already gone through that at this point?

Reply:
> I do have a small suggestion; would it be better for the documentation URL to point to the Quick Start page instead of the Installation page, given the user has already gone through that at this point?

good idea, I'll add one suggestion and check on linux

Reply:
works on linux

Reply:
@LostNeophyte Excellent news, thank you very much for confirming! I'm trying to get Windows setup at the moment.

Would you like me to change the documentation URL?

Reply:
Do as you think is best with the url, works either way for me.